src/Wordpress/CustomPostType/Timeframe.php

```diff
@@ -210,6 +209,7 @@ public function handleFormRequest()
 
     /**
      * Returns true, if timeframe is of type booking.
+     * TODO @markus-mw description seems not to be correct "Returns true, if timeframe is of type booking." seems that function name doesnt represent its real function?
      *
```

Comment:
No, it's right, but a little bit misleading. This function is the callback for the "show booking codes" CMB 2 field. ;) 
Anyway I'll rename it.